deploy/central/tasks-chart/templates/task-ods-build-go.yaml

```diff
@@ -50,6 +50,10 @@ spec:
         without leading `./` and trailing `/`.
       type: string
       default: "."
+    - name: cache-dependencies
+      description: Whether to cache dependencies in the PVC.
```

Comment:
Maybe this could explain that it will be the Go module cache, potentially linking to https://go.dev/ref/mod#module-cache?